src/vstart.sh

```diff
@@ -584,6 +584,12 @@ start_mon() {
 			--cap mgr 'allow *' \
 			"$keyring_fn"
 
+		prun $SUDO "$CEPH_BIN/ceph-authtool" --gen-key --name=client.fs --set-uid=0 \
+		   --cap mon 'allow r' \
+			--cap osd 'allow rw tag cephfs data=*' \
+			--cap mds 'allow rwp' \
+			"$keyring_fn"
```

Comment:
Let's also add `ceph fs authorize $fsname client.$fsname / rwp` for each file system we create?

Also, let's extend `fs authorize` to also take `*` as the file system name so we can also have the `client.fs` as above but without needing to use `ceph-authtool`. What do you think?

Reply:
Good suggestions. I've implemented them.

It turns out that * is a little awkward here because you don't have to quote the arguments to ceph fs authorize. So * gets expanded by the shell and a cryptic error message results that would be awkward to fix. So, should we:

1) Stay consistent and keep * as the wildcard, with documentation reminding users to escape it [did that, but still need doc and unit tests]
2) Find some other wildcard (like "all") that isn't error prone, but could conflict with a filesystem name

Thoughts @batrick and @gregsfortytwo ?